runtime/jcl/exports.cmake

```diff
@@ -637,7 +637,9 @@ if(J9VM_OPT_OPENJDK_METHODHANDLE)
 	if(JAVA_SPEC_VERSION EQUAL 8)
 		omr_add_exports(jclse Java_java_lang_invoke_MethodHandleNatives_getConstant)
 	endif()
-	if(JAVA_SPEC_VERSION GREATER_EQUAL 11)
+	# Only needed in Java version >= 11. Also, GREATER_EQUAL is not available in the
+	# minimum CMake version that OpenJ9 supports.
+	if(NOT JAVA_SPEC_VERSION LESS 11)
```

Comment:
I suggest these lines should just become
```
else()
```
There are several other instances of the pattern `NOT JAVA_SPEC_VERSION LESS n` earlier in this file (with matching comments); the comment isn't helpful.

Reply:
Updated.